community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```
@@ -0,0 +1,173 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments:
+      - name: hazelcast-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: hazelcast-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: hazelcast-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: hazelcast-operator
+                image: hazelcast/hazelcast-operator:0.0.1
+                imagePullPolicy: Always
+                name: hazelcast-operator
+                resources: {}
+              serviceAccountName: hazelcast-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
```

You are missing some RBAC to be allowed to create service accounts and roles/rolebindings. Please check these verbs are sufficient.

```suggestion
          - get
        - apiGroups:
          - ""
          resources:
          - serviceaccounts
          verbs:
          - get
          - create
          - list
          - update
          - delete
        - apiGroups:
          - rbac.authorization.k8s.io
          resources:
          - roles
          - rolebindings
          verbs:
          - get
          - create
          - list
          - update
          - delete
```